suite-native/module-accounts-import/src/accountsImportSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export const actionPrefix = '@accountsImport';
+
+export type AccountsImportState = Record<string, string>;
+
+const initialState: AccountsImportState = {};
+
+export type AccountsImportRootState = {
+    wallet: {
+        accountsImport: AccountsImportState;
+    };
+};
+
+const accountsImportSlice = createSlice({
```

Comment:
This funcionality has nothing to do with accounts import feature, because it's more something like accounts metadata that will be needed in many other packages. If you leave it here you will need to import this `module-accounts-import` package in every package where you will need a name of account. That's wrong because you shouldn't depend on accounts imports feature in packages that has nothing to do with it.

I would suggest new package called `accounts-metadata` where we will keep this kind of stuff related to accounts. Another option would be create package just called `metadata` where we will keep also other metadata related stuff because we will need labels for transactions, device names/icons...